fisher.fish

```diff
@@ -155,14 +155,20 @@ end
 function _fisher_self_update -a file
     set -l url "https://raw.githubusercontent.com/jorgebucaran/fisher/master/fisher.fish"
     echo "fetching $url" >&2
-    command curl -s "$url?nocache" >$file.
+    set -l curl_response (command curl -fsS "$url?nocache" -o $file. 2>&1)
+
+    if test $status -ne 0
+        echo "fisher: cannot update fisher -- $curl_response" >&2
+        command rm -f $file.
+        return 1
+    end
```

Comment:
@guihkx I think we could simplify it to this:

```fish
if not set -l curl_response (command curl -fsS "$url?nocache" -o $file. 2>&1)
    echo "fisher: cannot update fisher -- $curl_response" >&2
    command rm -f $file.
    return 1
end
```

Could you check again please?

Reply:
Done! Thanks for the tip.